ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -544,4 +530,127 @@ private boolean containsName(String name) {
       return Collections.unmodifiableCollection(idToAccountMap.values());
     }
   }
+
+  /**
+   * A {@link DataUpdater} to be used for updating {@link #FULL_ACCOUNT_METADATA_PATH} inside of
+   * {@link #updateAccounts(Collection)}
+   */
+  private class ZkUpdater implements DataUpdater<ZNRecord> {
+    private final Collection<Account> updatedAccounts;
+    private AccountInfoMap remoteAccountInfoMap;
+
+    /**
+     * @param updatedAccounts The {@link Account}s to update.
+     */
+    ZkUpdater(Collection<Account> updatedAccounts) {
```

Comment:
minor: would `accountsToUpdate` be a better name?

Reply:
yep, will change